test/e2e/common/pods.go

```diff
@@ -29,18 +29,18 @@ import (
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apimachinery/pkg/watch"
 	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 	"k8s.io/kubernetes/pkg/kubelet"
 	"k8s.io/kubernetes/test/e2e/framework"
+	imageutils "k8s.io/kubernetes/test/utils/image"
```

Comment:
Please move these changes to a separate PR (and copy me on it) so we can merge that first.  The only changes allowed in a conformance change are the ones that mark the tests conformance and the change to conformance.txt.

Reply:
Thanks for pointing it. Tracking changes in https://github.com/kubernetes/kubernetes/pull/69169